shared_infra/dynamo_write_heartbeat/src/dynamo_write_heartbeat.py

```diff
@@ -0,0 +1,26 @@
+# -*- encoding: utf-8 -*-
+"""
+Run a no-op write on DynamoDB (delete a non existent item) to be run in a scheduled lambda to
+make DynamoDB scale down when there would otherwise be zero throughput and it otherwise wouldn't.
+
+TABLE_NAMES: comma separated list of dynamoDb tables to call
+"""
+
+import boto3
+import os
+
+from wellcome_aws_utils.lambda_utils import log_on_error
+
+
+@log_on_error
+def main(event, context, dynamodb_client=None):
+    try:
+        table_names = [t.strip(' ') for t in os.environ['TABLE_NAMES'].split(',')]
+    except KeyError:
+        raise RuntimeError('TABLE_NAMES not found')
+
+    dynamodb_client = dynamodb_client or boto3.client('dynamodb')
+    for table_name in table_names:
+        response = dynamodb_client.delete_item(TableName=table_name.strip(),
+                                               Key={'id': {'S': 'dummy-id-not-there'}})
+        print("Heartbeat, delete_item on dynamoDb table:{}, returned-status:{}".format(table_name, response['ResponseMetadata']['HTTPStatusCode']))
```

Comment:
minor: it would be nice to make this a DEBUG log (I’ll have to check our other Lambdas for an example)

Reply:
Thanks, yes if you know how.  The log on error seems to use print.